auth0/src/main/java/com/auth0/android/authentication/AuthenticationAPIClient.kt

```diff
@@ -622,13 +565,12 @@ public class AuthenticationAPIClient @VisibleForTesting(otherwise = VisibleForTe
      * Fetch the token information from Auth0, using the authorization_code grant type
      * The authorization code received from the Auth0 server and the code verifier used
      * to generate the challenge sent to the /authorize call must be provided.
-     * <pre>
-     * `AuthenticationAPIClient client = new AuthenticationAPIClient(new Auth0("clientId", "domain"));
+     * ```
+     * val client = AuthenticationAPIClient(Auth0("clientId", "domain"))
```

Comment:
I noticed this one shows how to instantiate the client. We probably don't want this here, but maybe is useful to keep in the class constructor?

Reply:
New commit to remove instantiation example here, and add to constructor.